src/main/java/com/vi/counselingtoolsservice/api/controller/ProxyController.java

```diff
@@ -0,0 +1,133 @@
+package com.vi.counselingtoolsservice.api.controller;
+
+import com.vi.counselingtoolsservice.api.service.budibase.BudibaseProxyService;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.Enumeration;
+import javax.servlet.http.HttpServletRequest;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.client.HttpStatusCodeException;
+import org.springframework.web.client.RestTemplate;
+import org.springframework.web.util.UriComponentsBuilder;
+
+
+@RestController
+@Slf4j
+@RequiredArgsConstructor
+public class ProxyController {
```

Comment:
please add ProxyControllerIntegrationTest, ideally we should have some test that touches all layers